frontend/public/i18n.js

```diff
@@ -112,11 +112,11 @@ i18n
       },
       saveMissing: true,
       missingKeyHandler: function(lng, ns, key) {
-        window.windowError = new Error(
-          `Missing i18n key "${key}" in namespace "${ns}" and language "${lng}."`,
-        );
+        const error = new Error();
+        error.message = `Missing i18n key "${key}" in namespace "${ns}" and language "${lng}."`;
+        window.windowError = error;
         // eslint-disable-next-line no-console
-        console.error(`Missing i18n key "${key}" in namespace "${ns}" and language "${lng}."`);
+        console.error('Missing i18n key', error);
```

Comment:
I tried to `throw error` / `throw window.windowError`, but it doesn't get handled in [app.jsx](https://github.com/openshift/console/blob/ed2eb277b3d5bec45c2afda444f9644f2b38b189/frontend/public/components/app.jsx#L230) like Cypress does.
When I attempt to `throw` an error from `i18n.js`, protractor timesout with: 
```
(node:187007) UnhandledPromiseRejectionWarning: JavascriptError: javascript error: circular reference
  ...
  at Object.exports.checkErrors (/home/dtaylor/repos/openshift/origin/src/github.com/openshift/console/frontend/integration-tests/protractor.conf.ts:283:39)
(node:187007) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
```
I would've thought the `UnhandledPromiseRejection` would've been caught [here](https://github.com/openshift/console/blob/ed2eb277b3d5bec45c2afda444f9644f2b38b189/frontend/public/components/app.jsx#L235) in app.jsx.   Is `i18n.js` "out-of-scope" of `app.jsx`?   `window.windowError` existed in `protractor.conf.ts` so I assumed other window errors and unhandledrejections work/worked.